Fix any issues found when running the tests

Cruikshanks opened the pull request:
We have recently migrated a series of the legacy tests

- [Migrate abstraction alert tests](https://github.com/DEFRA/water-abstraction-acceptance-tests/pull/33)
- [Migrate paper return form tests](https://github.com/DEFRA/water-abstraction-acceptance-tests/pull/34)
- [Migrate address entry tests](https://github.com/DEFRA/water-abstraction-acceptance-tests/pull/36)

At the time though we were unable to run them. Now we can this change resolves any issues found. We also spotted we were using a `before()` rather than `beforeEach()` in some places. We favour a `beforeEach()` because what is in there does need to be done before each test. One day we hope to only need a `before()` block!

andrewhick commented:
Merging directly as this is a maintenance release to fix a bug in the suite. The test passes and so does Rubocop.

Cruikshanks committed:
Fix any issues found when running the tests

We have recently migrated a series of the legacy tests

- [Migrate abstraction alert tests](https://github.com/DEFRA/water-abstraction-acceptance-tests/pull/33)
- [Migrate paper return form tests](https://github.com/DEFRA/water-abstraction-acceptance-tests/pull/34)
- [Migrate address entry tests](https://github.com/DEFRA/water-abstraction-acceptance-tests/pull/36)

At the time though we were unable to run them. Now we can this change resolves any issues found. We also spotted we were using a `before()` rather than `beforeEach()` in some places. We favour a `beforeEach()` because what is in there does need to be done before each test. One day we hope to only need a `before()` block!
